modules/migration-excluding-resources.adoc

```diff
@@ -26,7 +26,7 @@ $ oc edit migrationcontroller <migration_controller> -n openshift-migration
 apiVersion: migration.openshift.io/v1alpha1
 kind: MigrationController
 metadata:
-  name: migration-controller
+  name: <migration-controller>
```

Comment:
@apinnick by default its name is `migration-controller`, nobody should modify it